test/localtest.sh

```diff
@@ -103,6 +103,26 @@ if [ -n "${EXPECTED_TOOL_VERSION}" ] && [ -z "$DD_NO_AGENT_INSTALL" ]; then
     fi
 fi
 
+if [ "${EXPECTED_FLAVOR}" = "datadog-agent" ] && [ -z "$DD_NO_AGENT_INSTALL" ]; then
```

Comment:
```suggestion
if [ "${EXPECTED_FLAVOR}" == "datadog-agent" ] && [ -z "$DD_NO_AGENT_INSTALL" ]; then
```

Reply:
I don't have a strong opinion on this, but this script is not POSIX compliant so we might as well use the bash niceties